modules/administration-guide/partials/proc_configuring-prod-cli-with-environment-variables-to-use-an-sftp-backup-server.adoc

```diff
@@ -0,0 +1,19 @@
+[id="configuring-prod-cli-for-an-sftp-backup-server-with-environment-variables_{context}"]
+= Configuring {prod-cli} for an SFTP backup server with environment variables
```

Comment:
Same as above: allow, those arguments, advanced grammar, and four VS one variables setup.